app/scripts/modules/core/snapshot/snapshot.restore.service.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.deck.core.snapshot.restore.service', [
```

Comment:
It feels a bit odd to have a service for a particular action, when most services in deck that make API calls are grouped into read and write services.